pkg/autoscaling/estimator/oom.go

```diff
@@ -47,7 +47,7 @@ func (e *OOMResourceEstimator) GetResourceEstimation(evpa *autoscalingapi.Effect
 		} else {
 			oomBumpUpRatio, err := strconv.ParseFloat(bumpUpRatio, 64)
 			if err != nil {
-				return nil, fmt.Errorf("Parse bumpUpRatio failed: %v. ", err)
+				return nil, fmt.Errorf("parse bumpUpRatio failed: %v. ", err)
```

Comment:
We already have golint in CI, what's the benefit to use reviewdog?

Or if we add some staticcheck config in Makefile?

WDYT @mfanjie